e2e/serial/dappInteractions/3_dappAccountsSwitcher.test.ts

```diff
@@ -230,7 +230,7 @@ describe.runIf(browser !== 'firefox')('Dapp accounts switcher flow', () => {
     expect(appConnectionRow).toBeTruthy();
   });
 
-  it('should be able to go back to extension, switch account and connect from nudge banner', async () => {
+  it.skip('should be able to go back to extension, switch account and connect from nudge banner', async () => {
```

Comment:
both tests are reenabled in https://github.com/rainbow-me/browser-extension/pull/2127 
the refactor temp brakes this test, but wheen the appSession store is migrated (mentioned PR) it works again